site/src/pages/components/accordion.explainer.mdx

```diff
@@ -0,0 +1,157 @@
+  - [CSS Toggles](#css-toggles)
+
+# Background
+
+A **disclosure** widget
+([research](/components/disclosure.research))
+is a UI that shows a summary or heading along
+with an indicator (sometimes a triangle or a plus sign) that allows the
+widget to be expanded to show additional details.
+An **accordion**
+([research](/components/accordion.research))
+is a sequence of related disclosure widgets.
+Some accordions are **exclusive accordions**,
+which means that at most one of the disclosures in that accordion
+can be open at the same time.
+
+The existing HTML
+[`details` element](https://html.spec.whatwg.org/multipage/C#the-details-element)
+represents a disclosure widget.
+A sequence of `details` elements can be used for a non-exclusive accordion.
+However, there is not a current web platform feature
+for exclusive accordions.
+
+It is worth noting that the HTML `details` element
+is not currently as styleable as it needs to be to meet many use cases.
+Improving styleability of `details` will be addressed in a separate proposal,
+intended to address both better ability to style/replace the disclosure triangle,
+and (maybe separately) to fit the parts of a `details`
+into different types of layout (such as grid).
+
+# Developer Demand for Exclusive Accordion
+
+Design systems that offer accordion widgets vary
+as to whether those widgets are exclusive or not.
+Based on the systems documented in the
+[accordion research](/components/accordion.research),
+it seems more common for them not to be exclusive,
+however, some systems offer an option for being exclusive,
+and some systems offer only exclusive accordions.
+
+Some examples of exclusive accordions on live sites:
+* [Pixel 7 product page](https://store.google.com/product/pixel_7?hl=en-US) on the Google Store (one accordion in each of the sections on Performance, Photography and Video, Protection, and Pixel Portfolio)
+* [Parsons Table product page](https://www.roomandboard.com/catalog/dining-and-kitchen/tables/parsons-tables) at Room and Board (Style/Size/Top/Base accordion on right side of page)
+* ...
+
+# Proposal
+
+I propose to add an attribute to
+[the `details` element](https://html.spec.whatwg.org/multipage/C#the-details-element).
+All `details` elements in the same tree
+that have the same value for this attribute
+would form an exclusive accordion.
+The syntax of this attribute and the rules for matching its values would match
+[the `name` attribute](https://html.spec.whatwg.org/multipage/C#naming-form-controls:-the-name-attribute)
+and its use in defining
+[radio button groups](https://html.spec.whatwg.org/multipage/C#radio-button-group).
+
+When these attributes are used to form a group, it means that
+any user action or API that causes one of the `details` elements
+in the group to open causes all other elements in the group to close.
+
+## Open design questions
+
+### Name
+
+I suggest that this attribute might be called `group`.
```

Comment:
Why not call it `name` to match the existing `name` attribute exactly? The explainer is ok as-is, I'm just curious why this wouldn't be the first choice.

Reply:
I changed it to be a little more neutral, but list `name` first.  I also mentioned that matching existing features could be an advantage or a disadvantage.